stdlib/option.mli

```diff
@@ -0,0 +1,75 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                                 OCaml                                  *)
+(*                                                                        *)
+(*   Copyright 2018 Institut National de Recherche en Informatique et     *)
+(*     en Automatique.                                                    *)
+(*                                                                        *)
+(*   All rights reserved.  This file is distributed under the terms of    *)
+(*   the GNU Lesser General Public License version 2.1, with the          *)
+(*   special exception on linking described in the file LICENSE.          *)
+(*                                                                        *)
+(**************************************************************************)
+
+(** Option values.
+
+    @since NEXT_RELEASE *)
+
+(** {1:options Options} *)
+
+type 'a t = 'a option
+(** The type for option values. Either [Some v] or [None]. *)
+
+val some : 'a -> 'a option
+(** [some v] is [Some v]. *)
```

Comment:
I know this is a bit useless, but having `Option.some` but not `Option.none` feels weird.